psi-ms.obo

```diff
@@ -22998,8 +22998,8 @@ relationship: has_units UO:0000010 ! second
 
 [Term]
 id: MS:4000052
-name: XIC-Height quartile ratios
-def: "The log ratio of successive XIC height quartiles. The metric's value triplet represents the log ratios of XIC-height-Q2 to XIC-height-Q1, XIC-height-Q3 to XIC-height-Q2, XIC-height max to XIC-height-Q3." [PSI:MS]
+name: obsolete XIC-Height quartile ratios
```

Comment:
Maybe not change the name by adding obsolete? The other obsolete term in the _.obo_  didn't go through a name change either as far as I can find.

Reply:
Leaving the names as-is is another layer of backwards compatibility, but it's not guaranteed the way the `id` is guaranteed to never change.

Reply:
I got these instructions from the Protege website for obsoleting terms, but happy to modify it according to conventions adopted by the PSI-MS CV.